modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/onboarding/context/index.jsx

```diff
@@ -70,15 +70,20 @@ const AppContextProvider = ({children}) => {
 					)
 					?.roleBriefs?.find(({name}) => name === 'Provisioning');
 
+				const isOmniAdmin = !!data.userAccount?.roleBriefs?.find(
+					({name}) => name === 'Administrator'
+				);
```

Comment:
In general, I like to use "Boolean" instead "!!" because of readability

```suggestion
				const isOmniAdmin = Boolean(data.userAccount?.roleBriefs?.find(
					({name}) => name === 'Administrator'
				));
```